### PR TITLE
[3.x] Fix when column name includes hyphen (-)

### PR DESCRIPTION
**Pull request by hnassr**

### CODE REVIEW

**Comment by Smolevich** [comment]
What is the root point for this PR?
Can you show bug or problem with code in master branch?

**Comment by Smolevich** [comment]
Failed test https://github.com/jenssegers/laravel-mongodb/pull/2127/checks?check_run_id=1275374233

**Comment by hnassr** [comment]
> What is the root point for this PR?
> Can you show bug or problem with code in master branch?

when i try this:

```php
$users = \App\Users::where('user-phone', 'LIKE', '%123%')->get();
```

i got this exception:

`MongoDB/Driver/Exception/CommandException with message 'ReferenceError: phone is not defined : @:1:15`

@Smolevich

**Comment by divine** [comment]
This fixes another "bug" that was added in https://github.com/jenssegers/laravel-mongodb/pull/2020

**Comment by divine** [comment]
What's the reason you've closed this PR?

**Comment by hnassr** [comment]
I think the problem occurred when the column name include hyphen like this `user-age`, so you cannot call it like this `this.user-age`, when i call it like this `this["user-age"]` the test run successfully.

**Review by gmedaglia — DISMISSED** [review]

**Review by Smolevich — DISMISSED** [review]

File: tests/QueryTest.php [hunk 1]
```diff
@@ -16,6 +16,7 @@ public function setUp(): void
         User::create(['name' => 'Brett Boe', 'age' => 35, 'title' => 'user']);
         User::create(['name' => 'Tommy Toe', 'age' => 33, 'title' => 'user']);
         User::create(['name' => 'Yvonne Yoe', 'age' => 35, 'title' => 'admin']);
+        User::create(['name' => 'Hazem Nassr', 'user-age' => 28, 'title' => 'member']);
```
Comment (divine) [inline tests/QueryTest.php:19]:
```suggestion
        User::create(['name' => 'John Smith', 'user-age' => 28, 'title' => 'member']);
```

I'm not happy with the fact you've added your name here, change it to something neutral.

**Review by divine — APPROVED** [review]

**Comment by divine** [comment]
Thank you for contribution!

**Pull request merged**